site/content/en/docs/tasks/run/plain_pods.md

```diff
@@ -16,7 +16,7 @@ This guide is for [batch users](/docs/tasks#batch-user) that have a basic unders
 
 ## Before you begin
 
-1. By default, the integration for `v1/pod` is not enabled.
+1. By default, the integration for `pod` is not enabled.
```

Comment:
Could you work on a separate PR so that we can immediately release this fix?

Reply:
Found other mentions of `v1/pod` while I was creating #4309

Reply:
Thanks!

Reply:
/lgtm
/approve